src/components/Palmid/hooks/useFasta.tsx

```diff
@@ -33,7 +33,7 @@ export function useFasta(): FastaHook {
     }
 
     useEffect(() => {
-        if (checkReportRequestCount >= 60) {
+        if (checkReportRequestCount >= 120) {
```

Comment:
It looks like there might be some caching in the backend so the error report is returned immediately. Is there any way to bypass that cache?

Reply:
I'm doing some retesting with the preview app link. Will post updates

Reply:
Alright, testing done! It turns out that there's a memory limit on the lambda API that kills the request. I also had to bump the memory limit on the API side. 

You can bypass the cache by change the name of the sequence. So instead of:
```
>betaflax_sativa
SNQFNDMKTLRGKNWNNYVYVFETIYPRHQSSDDLTFFAAIQKRLLRSNP
EKEARKLEKSWGIGSIMFHELKRTLGLNPNVYIDEEVVNREFVQKRLNKS
AKLIENHSGRSDPDWRLDHFFLFMKSQLCTKFEKRFVDAKAGQTLACFSH
QILTRFGVAFRTFEKKFSANLPKTWYVHTMKNFDQLNLWACENVKEREGT
ESDYEAFDRSQDAPILAFEILMLRFFNWPEDLIQDYKMIKLWMGCRLGAV
AIMRFTGEFGTFFFNTLVNMAFTVMRYHVNKESVIAFAGDDMYAAGFLKR
RVDLEFALDQLTLKAKVQFTRRPMFCGWYMTPMGIVKEPRLVLERWKIAE
QKGNLRDVIINYALEVSYGYRLGEYLWEVLDNLESQQEIVRNIVKSKHLL
PLKVRHFFSAVDNETLSTEF
```
try
```
>betaflax_sativa_test12345678
SNQFNDMKTLRGKNWNNYVYVFETIYPRHQSSDDLTFFAAIQKRLLRSNP
EKEARKLEKSWGIGSIMFHELKRTLGLNPNVYIDEEVVNREFVQKRLNKS
AKLIENHSGRSDPDWRLDHFFLFMKSQLCTKFEKRFVDAKAGQTLACFSH
QILTRFGVAFRTFEKKFSANLPKTWYVHTMKNFDQLNLWACENVKEREGT
ESDYEAFDRSQDAPILAFEILMLRFFNWPEDLIQDYKMIKLWMGCRLGAV
AIMRFTGEFGTFFFNTLVNMAFTVMRYHVNKESVIAFAGDDMYAAGFLKR
RVDLEFALDQLTLKAKVQFTRRPMFCGWYMTPMGIVKEPRLVLERWKIAE
QKGNLRDVIINYALEVSYGYRLGEYLWEVLDNLESQQEIVRNIVKSKHLL
PLKVRHFFSAVDNETLSTEF
```